backend/typescript/services/implementations/authService.ts

```diff
@@ -222,6 +222,29 @@ class AuthService implements IAuthService {
       return false;
     }
   }
+
+  async setPassword(
+    email: string,
+    newPassword: string,
+  ): Promise<ResponseSuccessDTO> {
+    let errorMessage = "An unknown error occured. Please try again later.";
+    try {
+      const uid = await (await firebaseAdmin.auth().getUserByEmail(email)).uid;
+      await firebaseAdmin.auth().updateUser(uid, {
+        password: newPassword,
+      });
+      return { success: true } as ResponseSuccessDTO;
+    } catch (error: any) {
+      Logger.error(`Failed to update password. Error: ${error}`);
+      if (error.code === "auth/invalid-password") {
+        errorMessage =
+          "Invalid password. Please make sure your password is at least 6 characters!";
```

Comment:
Wait how come here it's minimum 6 characters but in the frontend we make it minimum 8?